### PR TITLE
Updated README.md instructions for Debian builds

### PR DESCRIPTION
**Pull request by Devariiii**
While trying to install the .deb the program would just open and close instantly. After some trail and error I figured out that it was a dependency issue. 

I have updated the requirements.txt so it will download the correct packages (for Debian 12 at least) and updated the README.md instructions on how to install through said file. I have also added a note in case of .deb failure to try to download the dependencies manually.

### CODE REVIEW

**Comment by limburgher** [comment]
The requirements.txt file isn't a script, it's for use by pip to install modules locally.  Have you tried the deb builds on OBS?

https://build.opensuse.org/package/show/home:gwync/trelby

**Comment by Devariiii** [comment]
For the requirements.txt I didn't fully understand what was being done so I reverted my changes.

For the Debian package located at the OpenSuse repositories, it doesn't open unless also using `sudo apt install python3-setuptools wxpython-tools python3-lxml python3-reportlab`. Without that the program will instantly close as soon as it's opened. I've just added the step of installing for Debian packages before installation to the README.md file.

**Comment by limburgher** [comment]
Perfect, thank you!

**Review by flossybloom67 — CHANGES_REQUESTED** [review]
- **```
> `~~

~~`
```**

**Review by flossybloom67 — APPROVED** [review]

**Comment by flossybloom67** [comment]
3f0288fcad8777955fb41f7b40f8e2504628bc0e

File: README.md [hunk 1]
```diff
@@ -23,15 +23,19 @@ The major difference of this fork is conversion to Python 3.  I also have a list
 
 #### Debian and variants
```
Comment (flossybloom67) [inline README.md:24]: 3f0288fcad8777955fb41f7b40f8e2504628bc0e

**Review by flossybloom67 — APPROVED** [review]

**Pull request merged**